pkgs/by-name/ch/chicago95-gtk-theme/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
  stdenvNoCC,
  fetchFromGitHub,
}:

stdenvNoCC.mkDerivation rec {
```

Might be able to get away with `stdenvNoCC` if compilers aren't necessary. Would reduce rebuild frequency :slightly_smiling_face: